travel_platform/skyscanner_data/skyscanner_flights_data.py

```diff
@@ -0,0 +1,17 @@
+from skyscanner.skyscanner import Flights
+
+api_key = 'ka775911102027919433182126616312'
```

Comment:
API key should be part of configuration file.
See @kate-ka's pull request as an example.